releases_test.go

```diff
@@ -202,6 +315,33 @@ func Test_Releases(t *testing.T) {
 				if !strings.Contains(readmeContent, fmt.Sprintf("https://github.com/giantswarm/releases/tree/master/%s/%s", tc.provider, release.Name)) {
 					t.Errorf("expected link in README.md to %s release %s", tc.provider, release.Name)
 				}
+
+				// Check that all active releases contain all requested component versions
+				if release.Spec.State == "active" {
```

Comment:
This will cause issues if there are legacy releases which are active (e.g. 9.0.X) but you are actually just trying to create a new 11.X.X release. Not sure how to solve this here though.

Reply:
If your pattern matches both release 9.x and 11.x it's a dumb constraint and CI would have already complained when you added the request in the PR :smile: